### PR TITLE
feat(starfish): add transaction.op:http.server to webservice queries

### PR DESCRIPTION
**Pull request by DominikB2014**
I also double checked mobile, currently it does not use `useSpansQuery`, but instead uses traditional HOC, so no need to update anything there yet.

### CODE REVIEW

File: static/app/views/starfish/utils/useSpansQuery.tsx [hunk 1]
```diff
@@ -28,13 +28,15 @@ export function useSpansQuery<T = any[]>({
   enabled,
   referrer = 'use-spans-query',
   cursor,
+  view = 'webservice',
```
Comment (gggritso) [inline static/app/views/starfish/utils/useSpansQuery.tsx:31]: There's an enum for this! `StarfishType`

**Review by gggritso — COMMENTED** [review]
LGTM, just need to use an existing enum for the view type 👍🏻

File: static/app/views/starfish/utils/useSpansQuery.tsx (hunk 1 above)
Comment (DominikB2014) [reply static/app/views/starfish/utils/useSpansQuery.tsx:31]: Sick, didn't know that, thanks for pointing out, will update!

Comment (DominikB2014) [reply static/app/views/starfish/utils/useSpansQuery.tsx]: Made the changes!

**Review by gggritso — APPROVED** [review]
👍🏻

**Pull request merged**